components/accordion/src/tokens.json

```diff
@@ -228,48 +228,58 @@
             "type": "color"
           }
         },
-        "icon": {
-          "size": {
+        "focus-visible": {
+          "border-color": {
+            "$extensions": {
+              "nl.nldesignsystem.css.property": {
+                "syntax": "<color>",
+                "inherits": true
+              },
+              "nl.nldesignsystem.figma.supports-token": true
```

Comment:
@AliKdhim87 Als je nieuwe tokens toevoegt, dan moet in eerste instantie `...figma.support-token": false` gebruikt worden, tot Jeffrey or Rozerin de token in Figma ook heeft toegevoegd.

Zou je de tokens die nieuw zijn op `false` willen zetten?